README-trust.md

```diff
@@ -0,0 +1,116 @@
+Trust module
+============
+
+Description
+-----------
+
+The trust module allows to ensure presence and absence of a domain trust.
+
+Features
+--------
+* Trust management
+
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipagroup module.
+
+
+Requirements
+------------
+
+**Controller**
+* Ansible version: 2.8+
+
+**Node**
+* Supported FreeIPA version (see above)
```

Comment:
Samba 4 support, and server-trust-ad sub-package of IPA are also required on the node.

Reply:
I've implemented your suggestions